### DISCUSSION TITLE
Feature Request: Make it possible to search for entry by Uuid

### DISCUSSION THREAD

**Discussion by ar-ml** [keepassxreboot/keepassxc#11529] [Ideas]
If I enter a Uuid into search field it currently does not return anything.
Would be great to have this feature.

Usecase:
when using [cross references](https://keepassxc.org/docs/KeePassXC_UserGuide#_entry_cross_reference) in a very big database one might forget where the original entry is stored. I would like to be able to find the original entry by searching for the Uuid.

**Comment by droidmonkey** [comment]
https://github.com/keepassxreboot/keepassxc/pull/9571

Use `uuid:` search prefix

**Reply by michaelk83** [reply]
It's not in the user guide: https://keepassxc.org/docs/KeePassXC_UserGuide#_searching_the_database

**Reply by droidmonkey** [reply]
Yah just noticed that
